src/services/recim/team.ts

```diff
@@ -69,6 +69,10 @@ type PatchTeam = {
   Logo: string;
 };
 
+type TeamInviteResponse = {
+  Response: string;
+};
+
```

Comment:
Why is this it's own type with only one string proprty? Is it because that's how it is in the api? I believe I said we should just be passing a string in both places when the API PR came through.

Reply:
handled now as 
api/recim/team/{id}/invite/status
response is now in body